web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,78 @@
+# OCS Install Workflow
+
+### OperatorHub
+
+There will be 2 OCS operator entries in OperatorHub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
```

Comment:
It's not in 4.2. And there's no such thing as Ceph Cluster - it's OCS cluster.
However, we may want to support also a standalone RHCS cluster. Need to agree on that.